mmv1/products/vertexai/EndpointWithModelGardenDeployment.yaml

```diff
@@ -0,0 +1,985 @@
+# Copyright 2025 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+name: EndpointWithModelGardenDeployment
+api_resource_type_kind: Endpoint
+description: Create an Endpoint and deploy a Model Garden model to it.
+references:
+  guides:
+    "Use models in Model Garden": "https://cloud.google.com/vertex-ai/generative-ai/docs/model-garden/use-models"
+    "Overview of Model Garden": "https://cloud.google.com/vertex-ai/generative-ai/docs/model-garden/explore-models"
+    "Overview of self-deployed models": "https://cloud.google.com/vertex-ai/generative-ai/docs/model-garden/self-deployed-models"
+  api: "https://cloud.google.com/vertex-ai/docs/reference/rest/v1beta1/projects.locations/deploy"
```

Comment:
This should point at the GA docs rather than v1beta1

Reply:
Update to GA doc.